app/policies/timeline_event_policy.rb

```diff
@@ -21,7 +21,7 @@ def show?
     organisation = record.founders.first.user.organisation
 
     return false if organisation.blank?
-
-    user.organisations.exists?(id: record.user.organisation_id)
+    # check if the user is an admin of the organisation
```

Comment:
Do we need this comment?

Reply:
I thought this line `user.organisations.exists?(id: organisation.id)`, is somewhat confusing because of `user.organisations` & `user.organisation`.

Reply:
I actually agree with the comment being left in; this confused me a bit when I saw it recently.

@kaisersakhi You should follow standard English punctuation when writing comments, though. This should be:

```ruby
# Check if the user is an admin of the organisation.
```

Capitalize first letter. Put a full-stop after the sentence.